libs/api/domains/aircraft-registry/src/lib/aircraftRegistry.resolver.ts

```diff
@@ -0,0 +1,38 @@
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { CacheControl, CacheControlOptions } from '@island.is/nest/graphql'
+import { CACHE_CONTROL_MAX_AGE } from '@island.is/shared/constants'
+import { AircraftRegistryService } from './aircraftRegistry.service'
+import { AllAircraftsResponse } from './dto/allAircraftsResponse'
+import { AllAircraftsInput } from './dto/allAircrafts.input'
+import { AircraftsBySearchTermResponse } from './dto/AircraftsBySearchTermResponse'
+import { AircraftsBySearchTermInput } from './dto/aircraftsBySearchTerm.input'
+
+const defaultCache: CacheControlOptions = { maxAge: CACHE_CONTROL_MAX_AGE }
+
+@Resolver()
+export class AircraftRegistryResolver {
+  constructor(
+    private readonly aircraftRegistryService: AircraftRegistryService,
+  ) {}
+
+  @CacheControl(defaultCache)
```

Comment:
Should use `@CacheField()` in the object types if you want these results to be cacheable.

Reply:
I agree, it didn't pop up as a "lint error" so I simply forgot about that decorator, thanks for the reminder